plugins/org.jkiss.dbeaver.ext.cubrid/src/org/jkiss/dbeaver/ext/cubrid/edit/CubridViewManager.java

```diff
@@ -0,0 +1,28 @@
+/*
+ * DBeaver - Universal Database Manager
+ * Copyright (C) 2010-2023 DBeaver Corp and others
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.jkiss.dbeaver.ext.cubrid.edit;
+
+import org.jkiss.dbeaver.ext.generic.edit.GenericViewManager;
+
+public class CubridViewManager extends GenericViewManager {
+
+	@Override
+	public boolean canCreateObject(Object container) {
+		return false;
+	}

```

Comment:
Are views somehow created automatically in Cubrid, or the driver doesn't support creation?

Reply:
Cubrid support view creation.
I think the View creation is really similar to when we create view on SQL editor. So I disabled it and let user create view from SQL editor instead.
If you think it should be supported the View creation, I can enable it.